engine/movie/title.asm

```diff
@@ -1,4 +1,5 @@
 CopyDebugName:
+	; PrepareOakSpeech doesn't use this routine.
```

Comment:
So?

Reply:
I was going with an `; unused` comment at first, but it's technically used in `PrepareTitleScreen` despite being dead code. Should we note something here at all?

Reply:
Dead code is unused code, so yes, `; unused` is appropriate here. If you really wanted to be explicit, I'd go with:
```
CopyDebugName: ; unused except by dead code in PrepareOakSpeech
```
but IMO that's unnecessarily repeating what can be found by grepping for `CopyDebugName`. (It's understood that grepping will find something, since if it didn't the comment would be `; unreferenced`.)

Reply:
Just to clarify, would you prefer a simple `; unused` comment or none at all?

Reply:
Yes, just `; unused` please.